File: .audit/list-lang-keys.mjs
import fs from "node:fs";
const s = fs.readFileSync("src/i18n/translations.ts", "utf8");
const re = /\n  (\w+|"[^"]+"): \{/g;
const out = [];
let m;
while ((m = re.exec(s))) out.push(m[1]);
console.log(out.slice(0, 14).join(" | "));

File: .audit/list_lang_keys.py
# -*- coding: utf-8 -*-
import io, re
s = io.open('src/i18n/translations.ts', encoding='utf-8').read()
# find all language keys at top level of the translations object
for m in re.finditer(r'^\s{2}([a-zA-Z-]+):\s*\{', s, re.M):
    print(m.group(1))
